docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+
+```js {3}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+
+import App from './App'
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(<App />, rootElement)
+```
+
+In order to actually create a Redux store, we need to define our first "reducer" function.
+
+<Note type="success">
+
+#### Key Concept: "Reducers"
+
+A **_reducer_** is a function that know how to calculate a new state value when asked. A Redux store requires a single reducer function
+when it is created, which we call the "root reducer" to say that it is the starting point for any other reducer functions inside.
+
+Every reducer is called with two arguments: the current `state`, and an object called an **`action`**. We'll look at actions a bit later.
+
+Reducers must always follow certain rules inside. As a quick summary:
+
+- Reducers must always return a state value, never `undefined`
+- Reducers can only decide what the next state value should be by looking at the `state` and `action` arguments
+- Reducers are not allowed to make changes to the `state` argument, or anything else. They can only create the new state value by making copies of the originals, and updating the copies.
+
+</Note>
+
+For now, we can create a small root reducer function that always returns a string, like this:
+
+```js {7-9}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+
+import App from './App'
+
+function rootReducer(state, action) {
+  return 'Hello Redux!'
+}
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(<App />, rootElement)
+```
+
+Now we can create our first Redux store. We pass in the `rootReducer` function as a parameter. The store will call the reducer function,
+take whatever value it returns, and save that value inside as the current "state". We can then ask the store what its current state value is.
+
+```js {11-15}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+
+import App from './App'
+
+function rootReducer(state, action) {
+  return 'Hello Redux!'
+}
+
+const store = configureStore({
+  reducer: rootReducer
+})
+
+console.log(store.getState())
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(<App />, rootElement)
+```
+
+If you look at the console in your browser's DevTools (or in CodeSandbox), you should see that the words `"Hello Redux!"` were printed.
+
+### Set Up the Provider Component
+
+We've got a Redux store with its state inside, but now we need to find a way to use that state inside our `<App>` component.
+
+It's _possible_ that we could move the store into a separate file, import that store into the `App.js` file, and call `store.getState()`. But, as the
+app gets bigger, we'd have to import the store into _every_ file. That's not a good idea.
```

Comment:
I'm not sure how useful this information is to a beginner? Just comes off as more information than needed IMO

Reply:
Mmm... yes and no.  My thought was to justify why React-Redux's API is needed at all.

Reply:
I guess I'm just worried about this "Quick Start" page having more content than it needs, since in my mind the goal of this page is to provide as minimal information as possibly needed to get started with Redux and React.

If you feel like this content fits that goal then we should definitely keep it 😃

Reply:
I've seen a lot of beginners try to use vanilla Redux in React apps and they get very confused with how to handle the store. I agree with Mark here that it's important for us to explain why React Redux is helpful right away.

Of course, if we have a guide for non-React users in the future, this wouldn't be necessary.

Reply:
If this stays, it'd be good to explain _why_ "That's not a good idea." - it will get messy? hard to manage? but there's an easier way?